Fix #110 - add HedgeDoc info in extra (#184)
carpentries-incubator/jekyll-pages-novice: _extras/guide.md

## Introduction

> - We recommend the usage of [HedgeDoc](https://hedgedoc.org).

Reviewer: ```suggestion
- We recommend the usage of [HedgeDoc](https://hedgedoc.org) (formerly called CodiMD).
```

  All instructors can use it for note-taking when preparing or teaching this lesson and we also encourage

>   its usage during the workshop for asking questions and collaborate during  exercises, especially when using breakout rooms.

Reviewer: ```suggestion
  its usage during the workshop for asking questions and collaborate during exercises, especially when using breakout rooms.
```
extra space

  The main advantage is that it uses Markdown so it can easily convince learners on the importance of learning Markdown.

>   Make use of [HackMD Mechanics](https://coderefinery.github.io/manuals/hackmd-mechanics/#basic-controls) from [CodeRefinery](https://coderefinery.org/): it is important everyone understands how to use HedgeDoc from the very beginning of the workshop.

Reviewer: ```suggestion
  Make use of [HackMD Mechanics](https://coderefinery.github.io/manuals/hackmd-mechanics/#basic-controls) from [CodeRefinery](https://coderefinery.org/). Take time to orient your learners to using the document early on to be ensure everyone understands how to use HedgeDoc.
```
Small wording suggestion.
